packages/scoutgame-ui/src/components/common/NFTPurchaseDialog/components/NFTPurchaseForm.tsx

```diff
@@ -172,9 +171,13 @@ export function NFTPurchaseFormContent({ builder }: NFTPurchaseProps) {
     onError({ error, input }) {
       scoutgameMintsLogger.error('Error purchasing with points', { input, error, userId: user?.id });
       setSubmitError(error.serverError?.message || 'Something went wrong');
+      toast.success('Error while purchasing with points');
```

Comment:
i dont think we should show a toast here, there. is already an error inline in the form

Reply:
Done